src/core/libmaven/Fragment.h

```diff
@@ -95,4 +95,92 @@ class Scan;
         bool operator<(const Fragment* b) const;
         bool operator==(const Fragment* b) const;
     };
+
+    // TODO: from MAVEN (upstream). find our what this is.
+    struct FragmentationMatchScore {
```

Comment:
This is related to the Database feature we're going to add. When we match the fragmentation pattern from the MSP file and our samples, there will be a score calculated based on the extent of match. This is meant to store that information.